src/quo/components/wallet/amount_input/style.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo.components.wallet.amount-input.style
+  (:require
+    [quo.foundations.colors :as colors]
+    [react-native.platform :as platform]))
+
+(defn get-selection-color
+  [customization-color theme]
+  (colors/resolve-color customization-color
+                        theme
+                        (if platform/ios? 1 0.2)))
```

Comment:
Why is the value different for ios and Android here?

Reply:
That was a leftover from the copying of [this](https://github.com/status-im/status-mobile/pull/15717/files#diff-0dd59580d670c11f7451746ff026446453dffac5f17ac4782ed7e422e14f9921R32-R36) function. The `:selection-color` param is probably not needed here at all. What do you think @OmarBasem?

Reply:
@J-Son89 do you know why we have different values here?

Reply:
No idea, @nazariifenii whatever is in here should be matching the designs. So if it's not needed or just some mistakingly copied code then we probably don't need it.
We will ask the design team for a review of the component so once you feel it's at a ready state for that let us know and we can ask them :)

Reply:
If you need a hand inspecting the designs feel free to ask and we can go through it

Reply:
I would remove this prop. From Figma, I can see that we don't need different customizations of input selection and cursor colors. 
Also, I have access to hand inspecting the designs at Figma, so should be good, thank you!

Reply:
Removed in [this PR](https://github.com/status-im/status-mobile/pull/18687/commits/8430a37181a5be0cf27e84c1df525d5bcca9abc3).